src/dune_pkg/variable_value.ml

```diff
@@ -13,6 +13,9 @@ open! Import
 *)
 type t = String of string
```

Comment:
Not really related to this PR, but given there's just one constructor, maybe it can be removed?